app/views/registration_wizard/confirmation.html.erb

```diff
@@ -38,6 +38,11 @@
       <p class="govuk-body">
         <%= govuk_link_to "Read more about the Early Headship Coaching Offer", @form.ehco_more_information_url %>
       </p>
+    
+    <div class="govuk-inset-text">
+        <a target="_blank" href="https://docs.google.com/forms/d/e/1FAIpQLScAD7XvFlHl8LzfSyxstYErWMiFPXGamrA_qBhBJTGXuP9efw/viewform?usp=sf_link">Tell us about your experience</a> for the possibility to be called for further research and earn £40.
+      </div>
```

Comment:
Just adjusted the indentation a bit and added the `rel` attribute as suggested by the [design system guidance](https://design-system.service.gov.uk/styles/typography/#opening-links-in-a-new-tab).

```suggestion
    <div class="govuk-inset-text">
      <a target="_blank" rel="noreferrer noopener" href="https://docs.google.com/forms/d/e/1FAIpQLScAD7XvFlHl8LzfSyxstYErWMiFPXGamrA_qBhBJTGXuP9efw/viewform?usp=sf_link">Tell us about your experience</a> for the possibility to be called for further research and earn £40.
    </div>
```